### ISSUE TITLE
ImportError: No module named hulahop

### ISSUE THREAD

**Issue by yell0wfl4sh** [sugarlabs/baltigames-activity#2]
The error is logged on running this activity.

**Comment by mzeeqazi** [comment]
How to resolve such issue, ImportError Simply suggests that this module is not available in the system and need to be installed.

**Comment by Hrishi1999** [comment]
hulahop is deprecated. Any alternative solution @quozl ?

**Comment by quozl** [comment]
Yes, WebKit.  You have to get this activity working with Hulahop using a deprecated operating system so you can establish a baseline.  See also https://github.com/sugarlabs/baltigames-activity/issues/3.

**Comment by Hrishi1999** [comment]
@quozl Made changes here, but I am greeted with a white screen. Not sure what's, maybe an issue with WebKit2? https://github.com/hrishi1999/baltigames-activity

**Comment by quozl** [comment]
Thanks.  Reviewed https://github.com/Hrishi1999/baltigames-activity/commit/2b3a95080bc3abfab71b67fc35c386742465f434.

What did it used to look like?  i.e. what did you expect?

Is the WebView widget method `show()` called?  Perhaps the WebKit on your system cannot play these old Adobe Shockwave Flash files?  Try opening the SWF files with another WebKit-based browser, such as Epiphany.  Try substituting a static HTML file; that will prove if your GTK+ 3 port and WebKit port is working.

**Comment by Hrishi1999** [comment]
@quozl I didn't get to test the activity before the port because I don't have a "deprecated" system to test right now. When I launch the activity, its just white screen, can't even see the toolbar. Tried with Epiphany, the SWF won't load (missing plugin). Would try to debug in some time. (I tried to call the show() of WebKit2, no changes). Tried changing `class` to `exec` in activity.info too

**Comment by quozl** [comment]
You can install a deprecated operating system by downloading the archived ISO and running it in a virtual machine.  As you also use Windows, you could try loading the SWF file with Flash capable browsers.  That would at least test the file.

Install the Flash plugin; the SWF files require it.

I don't know which Linux distributions have good Flash plugin support, but in the case of Ubuntu the plugin has to be installed first.  This makes the activity less useful, but as it doesn't work _now_ and isn't on activities.sugarlabs.org, I'm not so concerned.

**Comment by tony37** [comment]
Bravo! The interns here in Rwanda just discovered this problem today 
trying various activities. I hope you can get this fixed.

Tony
